### PR TITLE
book list challenge

### PR DESCRIPTION
**Pull request by Jermaine-rbg**
Book list and NYC Transit Challenge

### CODE REVIEW

File: Jermaine/nyc_transit_challenge.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+'''
+class Station:
+    def __init__(self, station_name, location):
+        self.station_name = station_name
+        self.location = location
+    
+    def show_info(self):
+        print(f'{self.station_name} station is located at {self.location}')
+
+class subwaystation(Station):
```
Comment (Tylerlangsam) [inline Jermaine/nyc_transit_challenge.py:30]: Classes are always capitalized.

File: Jermaine/nyc_transit_challenge.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        self.open = False
+    
+    def show_info(self):
+        if open == True:
+            print(f'The station {self.station_name} located at {self.location} on the routes {self.routes} is open')
+
+        else:
+            print(f'The station is closed')
+
+Station_A = BusStation(("Penn Station","34th & 8th","Hudson River, New Jersey"))
```
Comment (Tylerlangsam) [inline Jermaine/nyc_transit_challenge.py:86]: No need for double parenthesis

**Review by Tylerlangsam — CHANGES_REQUESTED** [review]
Ive left a couple comments where changes need to be made, also it appears that you didn't do question 5 of the transit challenge

File: Jermaine/nyc_transit_challenge.py (hunk 2 above)
Comment (Jermaine-rbg) [reply Jermaine/nyc_transit_challenge.py:86]: On it

File: Jermaine/nyc_transit_challenge.py (hunk 1 above)
Comment (Jermaine-rbg) [reply Jermaine/nyc_transit_challenge.py:30]: Gotcha

**Comment by Jermaine-rbg** [comment]
I made the capitalization, removed the double () and I will push part 5 up. I will close this pull request and push up the correct changes.

**Pull request merged**